prac_05/emails.py

```diff
@@ -0,0 +1,27 @@
+def extract_name(email):
+    # Extract the name from the email by splitting at '@' and taking the first part
+    return email.split('@')[0].title()
```

Comment:
Seems like this would only return the first part of an email for example caleb.townsend@x would be returned as Caleb.townsend I believe. Taking in the prefix of the email as a variable and splitting it again at the . and rejoining our new 2 parts with a space using ' '.join looks like what you want here.

Reply:
Thank you, I have adjusted it to work for that however in my initial I did not as there are a few other ways of emails such as iestynTownsend@x or iestyn_towsnend@x so i had not accounted for cases where it was not simple fname@email